front/lib/swr/conversations.ts

```diff
@@ -365,6 +365,58 @@ export function useAddDeleteConversationTool({
   return { addTool, deleteTool };
 }
 
+export function useVisualizationRevert({
```

Comment:
this is not swr but other conversation related hooks are all here so I follow the pattern